packages/terra-tabs/src/Tabs.jsx

```diff
@@ -137,7 +137,15 @@ class Tabs extends React.Component {
     } = this.props;
 
     const commonTabItems = [];
-    const activeTabKey = this.state.activeKey || this.props.activeKey;
+    const tabKeys = [];
+
+    if (this.props.children && this.props.children.length > 0) {
+      this.props.children.forEach(child => {
+        tabKeys.push(child.key);
+      });
+    }
+    const foundActiveTabKey = tabKeys.find(element => element === this.props.activeKey);
+    const activeTabKey = foundActiveTabKey ? this.props.activeKey : this.state.activeKey;
```

Comment:
this can be simplified to : 
```suggestion
        const activeTab = React.Children.toArray(children).filter(child => child.key === this.props.activeKey);
        const activeTabKey = (activeTab) ? activeTab.key : this.state.activeKey
```

Reply:
Added.

Reply:
Curious !! why it didn't work.

Reply:
Using the above Suggestion, initially the active key is not getting set.

Reply:
Implemented the above suggested code.